deploy/README.md

```diff
@@ -83,6 +89,12 @@ To install infrastructure execute install.sh script as follows
 
 ```
 
+To enabled security features like Synapse managed VNET, managed private endpoints, and privates enpoints to 3 synapse endpoints, set 'securityEnabled=true' when running install.sh:
+```
+securityEnabled=true ./deploy/install.sh <environmentCode> <location> <envTag>
+```
+**Note that if you turn on securityEnabled during setup, Synapse endpoints are restricted to the custom VNET in the deployment environment. Thus, you need to create a Windows jumpbox inside the VNET to connect to Synapse Studio.**
```

Comment:
I think there is going to be a list of restrictions when we turn on this flag. Maybe create a separate section to list the restrictions when considering turning on the securityEnabled flag. We can keep this section updated as we go.